content/blog/2018/06/2018-06-26-new-api-token-system.adoc

```diff
@@ -0,0 +1,98 @@
+---
+:layout: post
+:title: "Security Hardening: New API token system in Jenkins 2.129+"
+:tags:
+- community
+- core
+- security
+- upgrade
+:author: wadeck
+---
+
+### About API tokens
+
+Jenkins API tokens are an authentication mechanism that allows a tool (script, application, etc.) to impersonate a user
+without providing the actual password for use with the Jenkins API or CLI.
+This is especially useful when your security realm is based on a central directory, like Active Directory or LDAP,
+and you don't want to store your password in scripts.
+Recent versions of Jenkins also make it easier to use the remote API when using API tokens to authenticate,
+as no CSRF tokens need to be provided even with CSRF protection enabled.
+API tokens are not meant to -- and cannot -- replace the regular password for the Jenkins UI.
+
+### Previous problems
+
+We addressed two major problems with the existing API token system in Jenkins 2.129:
+
+First, reported in link:https://issues.jenkins-ci.org/browse/JENKINS-32442[JENKINS-32442],
+user accounts in Jenkins have an automatically generated API token by default.
+As these tokens can be used to authenticate as a given user, they increase the attack surface of Jenkins.
+
+The second problem was reported in link:https://issues.jenkins-ci.org/browse/JENKINS-32776[JENKINS-32776]: 
+The tokens were previously stored on disk in an encrypted form.
+This meant that they could be decrypted by unauthorized users by leveraging another security vulnerability,
+or obtained, for example, from improperly secured backups, and used to impersonate other users.
+
+### New approach
+
+The main objective of this new system is to provide API tokens that are stored in an unidirectional way on the disk,
```

Comment:
an unidirectional -> a unidirectional
(because the `u` is a `y` sound in unidirectional, yay english)